tests/VoiceProfileClientTests.ts

```diff
@@ -91,16 +91,14 @@ test("VoiceProfileClient with Bad credentials throws meaningful error", (done: j
     objsToClose.push(r);
 
     const type: sdk.VoiceProfileType = sdk.VoiceProfileType.TextIndependentIdentification;
-    r.createProfileAsync( type, "en-us",
-        (res: sdk.VoiceProfile) => {
-            done.fail();
-        },
-        (error: string) => {
-            const expectedCode: number = 401;
-            const expectedMessage: string = "Access denied due to invalid subscription key or wrong API endpoint. Make sure to provide a valid key for an active subscription and use a correct regional API endpoint for your resource.";
-            expect(error).toEqual(`Error: createProfileAsync failed with code: ${expectedCode}, message: ${expectedMessage}`);
-            done();
-        });
+    try {
+        const res: sdk.VoiceProfile = await r.createProfileAsync(type, "en-us");
+    } catch (error) {
+        const expectedCode: number = 401;
+        const expectedMessage: string = "Access denied due to invalid subscription key or wrong API endpoint. Make sure to provide a valid key for an active subscription and use a correct regional API endpoint for your resource.";
+        expect(error.toString()).toEqual(`Error: createProfileAsync failed with code: ${expectedCode}, message: ${expectedMessage}`);
+        done();
+    }
```

Comment:
what if it does not throw at all for some reason?
Do you wnat to detect and fail for that?